.storybook/main.js

```diff
@@ -1,10 +1,25 @@
+const webpack = require('webpack')
+
 module.exports = {
-  "stories": [
-    "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx)"
-  ],
-  "addons": [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials"
-  ]
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx)'],
```

Comment:
Should we also add the `ts` and `tsx` extensions here? I think it'd be a good aim to use TypeScript for all our files